index.html

```diff
@@ -12376,6 +12608,14 @@ <h2>Relationship to RDF</h2>
       and <code>false</code>. The JSON-LD 1.1 Processing Algorithms and API specification [[JSON-LD11-API]]
       defines the <a data-cite="JSON-LD11-API#data-round-tripping">conversion rules</a>
       between JSON's native data types and RDF's counterparts to allow round-tripping.</li>
+    <li class="changed">As an extension to the <a data-cite="RDF11-CONCEPTS#data-model">RDF data model</a>,
+      <em>typed literals</em>, where the datatype is `xsd:string`, or <a>language-tagged strings</a>
```

Comment:
```suggestion
      <a>language-tagged strings</a>
```
I do not think that `xsd:string` should be mentionned here. This seems to imply that `{"@value": "hello", "@type": "xsd:string", "@direction": "ltr"}` is allowed, which is not the case.

Furthermore, in a direction-aware jsonld-to-rdf mapping, `{"@value": "hello", "@direction": "ltr"}` would *not* produce an`xsd:string`, but rather something in the line of `"hello"^^i18n:_ltr`.

Reply:
Alternative proposal, may be even more accurate:
```suggestion
      literals without an explicit datatype
```